dspace-api/src/main/java/org/dspace/content/clarin/ClarinUserRegistration.java

```diff
@@ -28,8 +28,12 @@
 @Table(name = "user_registration")
 public class ClarinUserRegistration implements ReloadableEntity<Integer> {
 
+    // Anonymous user
     public static final String ANONYMOUS_USER_REGISTRATION = "anonymous";
 
+    // Registered user without organization
+    public static final String UNKNOWN_USER_REGISTRATION = "Unknown";
```

Comment:
why capital?

Reply:
Because there is also a capital in v5, I'd like to keep it for consistency. 
![image](https://github.com/dataquest-dev/DSpace/assets/90026355/f934ba46-e3bf-489e-a68b-e9d59f414c63)